lib/CodeGen/TargetInfo.cpp

```diff
@@ -4276,6 +4276,15 @@ class PPC64_SVR4_ABIInfo : public ABIInfo {
 
   Address EmitVAArg(CodeGenFunction &CGF, Address VAListAddr,
                     QualType Ty) const override;
+
+  bool shouldPassIndirectlyForSwift( ArrayRef<llvm::Type*> scalars,
```

Comment:
Prevailing style in this project is UpperCamelCase variables. Also, there’s an extra space after the opening paren.